src/h5web/metadata-viewer/AttributeValueLink.tsx

```diff
@@ -0,0 +1,46 @@
+import { Fragment } from 'react';
+import { FiChevronsRight } from 'react-icons/fi';
+import styles from './MetadataViewer.module.css';
+
+interface Props {
+  onFollowPath: (value: string) => void;
+  value: unknown;
+}
+
+function AttributeValueLink(props: Props) {
+  const { onFollowPath, value } = props;
+
+  if (Array.isArray(value)) {
+    return (
+      <div className={styles.attrValueWrapper}>
+        [
+        {value.map((val, i) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <Fragment key={i}>
+            <AttributeValueLink onFollowPath={onFollowPath} value={val} />
+            {i < value.length - 1 && <span>,</span>}
+          </Fragment>
+        ))}
+        ]
+      </div>
+    );
+  }
+
+  if (typeof value === 'string' && value !== '.') {
```

Comment:
`.` is a placeholder and cannot be used for navigation so we might as well put it out of the way.